src/js/components/login/login.js

```diff
@@ -84,7 +84,7 @@ export const EntryLink = ({ className = '', target = 'signup' }) => (
 );
 
 export const LocationWarning = () => {
-  const location = Object.entries(locations).reduce((accu, [key, value]) => (window.location.hostname.includes(value.location) ? key : accu), locations.us.key);
+  const location = Object.entries(locations).reduce((accu, [key, value]) => (window.location.hostname == value.location ? key : accu), locations.us.key);
```

Comment:
having a list of locations breaks other parts though... but:

```suggestion
  const location = Object.entries(locations).reduce((accu, [key, value]) => ([`staging.${value.location}`, value.location].includes(window.location.hostname) ? key : accu);
```

should get us the staging support...

Reply:
I believe we need a start value for the reduce function to make it work.